modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/DefaultVariant.es.js

```diff
@@ -67,7 +68,9 @@ export const Column = forwardRef(
 		return (
 			<ClayLayout.Col
 				{...addr}
-				className="col-ddm"
+				className={classnames('col-ddm', {
+					hide: firstField?.hideField && !editable,
+				})}
```

Comment:
@mateusgomes01, move this logic to WizardVariant, please :)

Reply:
Done. See it here https://github.com/liferay-forms/liferay-portal/pull/484/commits/0a4426a3b02ae10b2e3a48b798a84d29f113df14